.gitignore

```diff
@@ -16,6 +16,7 @@
 npm-debug.log*
 yarn-error.log
 testem.log
+package-lock.json
```

Comment:
Lockfiles should be committed. We use yarn here, but having both would be fine.

Reply:
gonna revert back, since there was none in the project i thought you guys didn't use them.

Reply:
There is a yarn.lock, as I typically use yarn, but shouldn't hurt to have both.